ccmlib/node.py

```diff
@@ -72,7 +72,7 @@ class Node(object):
     Provides interactions to a Cassandra node.
     """
 
-    def __init__(self, name, cluster, auto_bootstrap, thrift_interface, storage_interface, jmx_port, remote_debug_port, initial_token, save=True, binary_interface=None):
+    def __init__(self, name, cluster, auto_bootstrap, thrift_interface, storage_interface, jmx_port, remote_debug_port, byteman_port, initial_token, save=True, binary_interface=None):
```

Comment:
Could you make byteman_port a kwarg that defaults to 0?

`byteman_port=0`

Anything accessing the node.py API directly will start failing now, otherwise. The same with the dse_node header, please.